readme.md

```diff
@@ -17,11 +17,15 @@ $ npm install --save ansi-styles
 ## Usage
 
 ```js
-const ansi = require('ansi-styles');
+const style = require('ansi-styles');
 
-console.log(ansi.green.open + 'Hello world!' + ansi.green.close);
-```
+console.log(style.green.open + 'Hello world!' + ansi.green.close);
 
+// color conversion between 16/256/16 million colors
+console.log(style.bgColor.ansi.hsl(120, 80, 72) + 'Hello world!' + style.bgColor.close);
```

Comment:
Because chalk will need to know this for degrading colors down as well in the event `supports-color` returns `1`.